MIGRATING.md

```diff
@@ -1,7 +1,7 @@
 # Migrate from the SignalFx Tracing Library for Ruby
 
-The SignalFx Tracing Library for Ruby is deprecated. Replace it with the
-agent from the Splunk Distribution of OpenTelemetry Ruby.
+The SignalFx Tracing Library for Ruby is scheduled to be deprecated by the end of 2022.
+Replace it with the agent from the Splunk Distribution of OpenTelemetry Ruby.
```

Comment:
I don't think we should use the term "agent" here and in other places in this document that aren't part of this PR. Just call it the Distribution or a library?